### ISSUE TITLE
FEATURE - Special Professions List

### ISSUE THREAD

**Issue by mike10101011** [Hoizame/AtlasLootClassic#66]
**Is your feature request related to a problem? Please describe.** 
No

**Describe the solution you'd like**
It'd be nice to filter out specialty professions. 

**Additional context**
For example, in Leatherworking, There are filters for Leather or Mail and then filters for Back, Chest, Feet, etc. Can you add a filter for Dragonscale, Tribal, Elemental, and then whichever the other specialties are for the other professions?

**Comment by Hoizame** [comment]
hmm yes I think about what's the best way to implement this

**Comment by mike10101011** [comment]
I hope the two files attached properly. I took a look at the code. I believe the change is pretty simple. The PNG is my end result on adding Dragonscale to the list for Leatherworking. The TXT is the code I used. I just added it at the end of the Leatherworking section after Misc (in data.lua). I believe if this were to be added for each of the specialties, it would be a great addition to your add-on. 

Also, I checked the leather chest, leather feet, etc. and the Dragonscale pieces are still listed, so adding another sub-section does not affect the others. 

Only other thing to note, I just listed the material as Normal rather than Mail since there's only one material used for this specialty. 

![Dragonscale](https://user-images.githubusercontent.com/60323260/73141941-f2999680-404e-11ea-8a37-b2502dd7b212.PNG)
[Dragonscale.txt](https://github.com/Hoizame/AtlasLootClassic/files/4114385/Dragonscale.txt)
